dynatrace-operator-google-marketplace/data-test/schema.yaml

```diff
@@ -0,0 +1,30 @@
+properties:
+  name:
+    type: string
+    default: "dynatrace-operator"
+    x-google-marketplace:
+      type: NAME
+  namespace:
+    type: string
+    default: "dynatrace"
+    x-google-marketplace:
+      type: NAMESPACE
+  apiUrl:
+    type: string
+    description: The Dynatrace api url including `/api` path at the end
+    default: "https://mydynatraceenvironmenttesturl.com/api"
```

Comment:
Although could you please change the URL to something that is not really an URL.

Technically speaking, somebody could come and register this domain, and now someone is running an privileged Pod for testing running code from some random guy.